Pearson-Advance/frontend-app-instructors-portal
# feat: header and footer

# Ticket 
https://agile-jira.pearson.com/browse/PADV-1171


## Description
In this PR is added the header and footer using  [https://www.npmjs.com/package/react-paragon-topaz](https://www.npmjs.com/package/react-paragon-topaz)

## Screenshots

![f](https://github.com/Pearson-Advance/frontend-app-instructors-portal/assets/52179095/4333f80b-2b5b-445c-b0b2-2480d6877075)


## How to test
- Clone the instructor Portal MFE
- Run openedx services, make sure to configure a site in the port 1990 with the proper settings.
- Run 
```bash
npm install
```
- Then
```bash
npm run start
```
- Go to [http://http://localhost:1990](http://http//localhost:1990)
